src/main/resources/com/openshift/jenkins/plugins/OpenShiftDSL.groovy

```diff
@@ -298,7 +298,7 @@ class OpenShiftDSL implements Serializable {
                 context.setServerCertificateAuthorityContent(cc.getServerCertificateAuthority());
                 context.setCredentialsId(cc.credentialsId);
                 context.setProject(cc.defaultProject);
-                context.setServerUrl(cc.getServerUrl(), cc.isSkipTLSVerify());
+                context.setServerUrl(cc.getServerUrl(), cc.isSkipTlsVerify());
```

Comment:
how was this not broken before?

Reply:
@bparees Sorry, looks like a pull and not a jenkins changed my environment. Looks like it broke 21 days ago: https://github.com/openshift/jenkins-client-plugin/blame/master/src/main/resources/com/openshift/jenkins/plugins/OpenShiftDSL.groovy#L301

Reply:
@coreydaley @gabemontero fyi.  Hopefully this is not part of the jenkins v3.7 client plugin version?

Reply:
OK, I can confirm that this is *NOT* in the v1.0.2 of the client plugin, which is what when out with 3.7 of the openshift jenkins rhel image.

Whew :-)

The particulars:
- the commit for https://github.com/openshift/jenkins-client-plugin which introduced this change is fbcbdbac035a6017c3eb71526e08a5a46af8c2f0
- I have not cherry-picked that commit into https://github.com/jenkinsci/openshift-client-plugin yet
- And as you can see from https://github.com/jenkinsci/openshift-client-plugin/blob/master/src/main/resources/com/openshift/jenkins/plugins/OpenShiftDSL.groovy#L304 it is still `isSkipTlsVerify`.

As for the reversion of the name change during the PR review, it appears the reference if the groovy file is what got us (since that does not show up in compiles).  It won't show up as a compile failure (any references), and whatever manual testing that was done simply missed it.  I know that @coreydaley and I also had some conversations around updating the automated testing around several of the items he worked on (where we can't trigger the test until after the centos image is updated).  I'll let him chime in if the insecure tls was one of those items on his radar.  And if not, let's add it.